front-end/src/components/BaseButton.jsx

```diff
@@ -51,10 +52,20 @@ BaseButton.propTypes = {
     size: PropTypes.string,
     isDisabled: PropTypes.bool,
     icon: PropTypes.object,
+    label: function(props) {
```

Comment:
(another nit for follow up. Assume all nits are follow up PR stuff actually): arrow function please :pleading_face: